vol_misc_test.c

```diff
@@ -23,31 +23,29 @@ static int test_double_init_term(void);
  * The array of miscellaneous tests to be performed.
  */
 static int (*misc_tests[])(void) = {
-        test_open_link_without_leading_slash,
-        test_object_creation_by_absolute_path,
-        test_absolute_vs_relative_path,
-        test_dot_for_object_name,
-        test_symbols_in_compound_field_name,
-        test_double_init_term,
+    test_open_link_without_leading_slash, test_object_creation_by_absolute_path,
+    test_absolute_vs_relative_path,       test_dot_for_object_name,
+    test_symbols_in_compound_field_name,  test_double_init_term,
```

Comment:
Uglier

Reply:
Yeah I think this is definitely one annoying habit of clang-format. Sure, if you have a lot of tests it'll start to take up vertical space, but having two on a line makes it harder to find what you're looking for.